c2corg_ui/static/js/documentediting.js

```diff
@@ -359,16 +361,23 @@ app.DocumentEditingController.prototype.handleMapFeatureChange_ = function(
     feature) {
   var geometry = feature.getGeometry();
   goog.asserts.assert(geometry);
+  var isPoint = geometry instanceof ol.geom.Point;
   var data = this.scope_[this.modelName_];
   // If creating a new document, the model has no geometry attribute yet:
   data['geometry'] = data['geometry'] || {};
-  data['geometry']['geom'] = this.geojsonFormat_.writeGeometry(geometry);
-  if (geometry instanceof ol.geom.Point) {
-    var coords = this.getCoordinatesFromPoint_(geometry.clone());
+  if (isPoint) {
+    data['geometry']['geom'] = this.geojsonFormat_.writeGeometry(geometry);
+    var coords = this.getCoordinatesFromPoint_(
+        /** @type {ol.geom.Point} */ (geometry.clone()));
     data['lonlat'] = {
       'longitude': coords[0],
       'latitude': coords[1]
     };
+  } else {
+    var center = ol.extent.getCenter(geometry.getExtent());
+    var centerPoint = new ol.geom.Point(center);
```

Comment:
True, but it only works with ``LineString`` I think. And since we get ``MultiLineString``... :P Or maybe we should use the middle point of the first ``LineString``?
I have added a fixup for this: 6d0dc2898f86b7d3f8b3434e1b1b8a5e8a55364a but I wonder if we should not use a more consistent approach for LineStrings/MultiLineStrings.